### DISCUSSION TITLE
求麦本本锋麦s的efi配置文件

### DISCUSSION THREAD

**Discussion by lemonstable** [daliansky/Hackintosh#386] [Q&A]
机型稍微有些老，但还够用求文件

处理器	英特尔 第四代酷睿 i7-4720HQ @ 2.60GHz 四核
主板	Standard To be filled by O.E.M.
内存	8 GB ( 金士顿 DDR3L 1600MHz )
主硬盘	三星 MZNTE128HMGR-00000 ( 128 GB / 固态硬盘 )
主显卡	Nvidia GeForce GTX 960M ( 2 GB )
显示器	LG LGD0470 ( 15.7 英寸  )
网卡	英特尔 Dual Band Wireless-AC 3160
声卡	瑞昱 ALC269 @ 英特尔 Lynx Point  高保真音频

**Comment by nabaonan** [comment]
cpu老了，新系统够呛

**Comment by merrystars** [comment]
您好，我已收到您的邮件。
